site/content/docs/5.1/examples/carousel/carousel.rtl.css

```
@@ -49,6 +49,11 @@ body {
   margin: 5rem 0; /* Space out the Bootstrap <hr> more */
 }
 
+/* Thin out the marketing headings */
+.featurette-heading {
+  letter-spacing: -.05rem;
+}
+
```

RTLCSS block syntax should do the trick:

```suggestion
/* Thin out the marketing headings */
/* rtl:begin:remove */
.featurette-heading {
  letter-spacing: -.05rem;
}
/* rtl:end:remove */

```